packages/graphback-codegen-client/src/templates/gqlTemplates.ts

```diff
@@ -67,7 +57,7 @@ export const createMutation = (t: GraphQLObjectType) => {
 export const updateMutation = (t: GraphQLObjectType) => {
   const fieldName = getFieldName(t.name, GraphbackOperationType.UPDATE)
 
-  return `mutation ${fieldName}($input: ${t.name}Input!) {
+  return `mutation ${fieldName}($input: Update${t.name}Input!) {
```

Comment:
We have helper for this https://github.com/aerogear/graphback/blob/52cacda4f03dafd5fc1b70a9d2aaf689669fdad5/packages/graphback-core/src/crud/mappingHelpers.ts#L54